reference/marketing.v2.yml

```
@@ -1621,8 +1622,8 @@ components:
             example: Mon, 19 Jan 1970 07:21:46 CST
           expiry_date:
             type: string
-            description: Date on which the gift certificate is set to expire.
-            example: "1622583106"
+            description: Date on which the gift certificate is set to expire. The date must be in [RFC 2822](http://tools.ietf.org/html/rfc2822#section-3.3) format.
```

```suggestion
            description: Date on which the gift certificate is set to expire. The date must be in [RFC 2822](https://www.rfc-editor.org/rfc/rfc2822#section-3.3) format.
```
link redirected here.